test/integration/hooks.spec.js

```diff
@@ -42,10 +42,10 @@ describe('hooks', function() {
       ];
 
       expected.forEach(function(line, i) {
-        assert.equal(lines[i], line);
+        assert.strictEqual(lines[i], line);
```

Comment:
Is there a reason that you keep `assert` rather than `unexpected`.

Reply:
too lazy to use `expect`.  these need to be converted.